encrypt_cert.sh

```diff
@@ -16,14 +16,19 @@ subnet_ip=$(($(cat /etc/nebula/client_subnet.number) + 1))
 echo $subnet_ip > /etc/nebula/client_subnet.number
 
 #create nebula cert and keys
-mkdir $1_nebula_cert
+mkdir /tmp/$1_nebula_cert
```

Comment:
consider using `mktemp`, e.g. `tmpdir=$(mktemp -d)`

Reply:
Uniqueness of the filename is ensured by the use of the requested name. And there doesn't seem to be a security benefit to mktemp. Is this just because it's custom to use mktemp for files in `/tmp`?